app/services/proofing/lexis_nexis/response.rb

```diff
@@ -31,6 +31,12 @@ def reference
       # @api private
       def response_body
         @response_body ||= JSON.parse(response.body)
+      rescue JSON::ParserError => exception
+        # IF a JSON parse error occurs the resulting error message will contain the portion of the
+        # response body where the error occured. This portion of the response could potentially
+        # include sensitive informaiton. This commit scrubs the error message by raising a JSON
+        # parse error with a generic message.
+        raise JSON::ParserError, 'An error occured parsing the response body JSON'
```

Comment:
should/can we include the response status code and response as hints of what happened?
```suggestion
        raise JSON::ParserError, "An error occured parsing the response body JSON, status=#{response.status} content_type=#{response.content_type}"
```

Reply:
Yeah, it probably doesn't hurt